addons/mail/populate/mail_message.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+from odoo.tools import populate
+
+
+class Message(models.Model):
+    _inherit = "mail.message"
+    _populate_dependencies = ["res.partner"]
+
+    _populate_sizes = {
+        'small': 1e4,
+        'medium': 1e7,
+        'large': 1.5e8,
+    }
+
+    def _populate_factories(self):
+        partner_ids = self.env.registry.populated_models['res.partner']
+
+        def get_res_id(values, counter, random):
+            if values['model'] == 'res.partner':
+                return random.choice(partner_ids)
+            return False
```

Comment:
res_id should usually not be False in messages.
I think we have some exceptions, but I wouldn't mind if all populate messages had a defined thread.